webserver/pkgpkr/static/styles.css

```diff
@@ -62,6 +83,15 @@ html {
 body {
   height: 100%;
   overflow: scroll;
+  scrollbar-width: none; /* Hide scrollbar for Firefox */
```

Comment:
I wish there was a cleaner way to get rid of the scrollbar, but I assume this was the best way.

Reply:
when I googled things, many threads says "but FireFox does not support this method anymore" so I had to do this for FireFox :( let me see if there is other way to do that